script.js

```diff
@@ -0,0 +1,111 @@
+    document.getElementById('hamburger_button_x').style.display = "block"
+    document.getElementsByTagName('body')[0].style.overflow = 'hidden'
+    is_on = true
+    }
+})
+
+function close_button() {
+    if (screen.width < 768) {
+    document.getElementById('item_nav').style.display = 'none'
+    document.getElementById('header').style.display = 'none'
+    document.getElementById('hamburger_button_x').style.display = "none"
+    document.getElementsByTagName('body')[0].style.overflow = 'scroll'
+    is_on = false
+    }
+}
+
+
+const card_info = {
+    image: ["./assets/1Placeholder.svg", "./assets/2ndPlaceholder.svg", "./assets/3rdPlaceholder.svg", "./assets/1Placeholder.svg", "./assets/2ndPlaceholder.svg", "./assets/3rdPlaceholder.svg", "./assets/Snapshoot Portfolio.svg", "./assets/Img Placeholder.svg"],
+    name: ["Profesional Art Printing Data More", "Data Dashboard Healthcare", "Website Protfolio", "Profesional Art Printing Data More", "Data Dashboard Healthcare", "Website Protfolio", "Multi-Post Stories"],
+    description: ["A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard","A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard","A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard","A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard","A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard","A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard", "A daily selection of privately personalized reads; no accounts or sign-ups required. has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a standard dummy text."],
+    live_link: ["https://rawcdn.githack.com/glitcher255/micro-porfolio/3aef2ff8602559c098666dcf4522e7baa82ca27c/index.html","https://rawcdn.githack.com/glitcher255/micro-porfolio/3aef2ff8602559c098666dcf4522e7baa82ca27c/index.html","https://rawcdn.githack.com/glitcher255/micro-porfolio/3aef2ff8602559c098666dcf4522e7baa82ca27c/index.html","https://rawcdn.githack.com/glitcher255/micro-porfolio/3aef2ff8602559c098666dcf4522e7baa82ca27c/index.html","https://rawcdn.githack.com/glitcher255/micro-porfolio/3aef2ff8602559c098666dcf4522e7baa82ca27c/index.html","https://rawcdn.githack.com/glitcher255/micro-porfolio/3aef2ff8602559c098666dcf4522e7baa82ca27c/index.html"],
+    source_link: ["https://github.com/DavidCazco36/My-Portfolio", "https://github.com/DavidCazco36/My-Portfolio", "https://github.com/DavidCazco36/My-Portfolio", "https://github.com/DavidCazco36/My-Portfolio", "https://github.com/DavidCazco36/My-Portfolio", "https://github.com/DavidCazco36/My-Portfolio"],
```

Comment:
- [ OPTIONAL ] I think for the two buttons in your mobile version you should give them the right width since I can see they have more height in your design so if you give the buttons more width it would be great to make your design look like the Figma 🚀💯✨

- your design
![image](https://user-images.githubusercontent.com/58553711/209029005-f8bdaf74-ce59-443c-90e1-164740d7de1f.png)

- Figma
![image](https://user-images.githubusercontent.com/58553711/209029061-96ba77ee-755e-4bda-8376-fee59ebde72e.png)

Reply:
Its 3 am and my partner is sleeping, but surely Ill fix it someday, thank you Reem 👍👍👍